### ISSUE TITLE
[Bug] ERROR: Invalid requirement: 'openvino-dev[tensorflow2,

### ISSUE THREAD

**Issue by nyck33** [openvinotoolkit/openvino#11619]
##### System information (version)

- OpenVINO => (from conda list)
openvino                  2022.1.0                 pypi_0    pypi
openvino-dev              2022.1.0                 pypi_0    pypi
openvino-telemetry        2022.1.1                 pypi_0    pypi

- Operating System / Platform => Ubuntu 20.04 LTS
- Python
- Problem classification: Pypi installation
- Framework: none
- Model name: none
-->

- OpenVINO=> :grey_question:
- Operating System / Platform => :grey_question:
- Compiler => :grey_question:
- Problem classification => :grey_question:

##### Detailed description
I want the C++ and Python versions but saw this:
'Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).'
here: https://docs.openvino.ai/nightly/openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies
'
Does that really mean installing from PyPI makes those tools available for C++?  
I just have never encountered a cross-language installation.

So I was able to successfully install and run a sample Jupyter notebook following instructions here: https://github.com/openvinotoolkit/openvino_notebooks/wiki/Ubuntu

But I am guessing to do any development I need the install that is causing the error in the subject line.

Firstly, is using a conda environment rather than venv wrong?

##### Steps to reproduce

I'll go back and try to use venv.

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
 
   Places to check:
   * OpenVINO documentation: https://docs.openvino.ai/
   * OpenVINO forum: https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/bd-p/distribution-openvino-toolkit
   * OpenVINO issue tracker: https://github.com/openvinotoolkit/openvino/issues?q=is%3Aissue
   * Stack Overflow branch: https://stackoverflow.com/questions/tagged/openvino
   -->
 - [ ] There is reproducer code and related data files: images, videos, models, etc.
   <!--
   The best reproducer -- test case for OpenVINO that we can add to the library.

**Comment by nyck33** [comment]
I tried without parameters
```
(openvino_env) nobu@nobu-IdeaPad-3-15IIL05:/media/nobu/NTFSUbuWin/openvino/openvino_notebooks$ pip install openvino-dev
Requirement already satisfied: openvino-dev in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (2022.1.0)
Requirement already satisfied: addict>=2.4.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2.4.0)
Requirement already satisfied: opencv-python==4.5.* in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (4.5.5.64)
Requirement already satisfied: scipy~=1.5.4 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.5.4)
Requirement already satisfied: parasail>=1.2.4 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.2.4)
Requirement already satisfied: networkx~=2.6 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2.8)
Requirement already satisfied: pydicom>=2.1.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2.3.0)
Requirement already satisfied: requests>=2.25.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2.27.1)
Requirement already satisfied: nibabel>=3.2.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (3.2.2)
Requirement already satisfied: py-cpuinfo>=7.0.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (8.0.0)
Requirement already satisfied: fast-ctc-decode>=0.2.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.3.2)
Requirement already satisfied: defusedxml>=0.7.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.7.1)
Requirement already satisfied: jstyleson~=0.0.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.0.2)
Requirement already satisfied: sentencepiece>=0.1.95 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.1.96)
Requirement already satisfied: pandas~=1.1.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.1.5)
Requirement already satisfied: tokenizers~=0.10.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.10.3)
Requirement already satisfied: transformers>=4.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (4.16.2)
Requirement already satisfied: numpy<=1.21,>=1.16.6 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.19.5)
Requirement already satisfied: openvino-telemetry>=2022.1.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2022.1.1)
Requirement already satisfied: pyyaml>=5.4.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (6.0)
Requirement already satisfied: pyclipper>=1.2.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.3.0.post2)
Requirement already satisfied: shapely>=1.7.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.8.2)
Requirement already satisfied: openvino==2022.1.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2022.1.0)
Requirement already satisfied: lmdb>=1.2.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.3.0)
Requirement already satisfied: scikit-image>=0.17.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.19.2)
Requirement already satisfied: pillow>=8.1.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (9.1.0)
Requirement already satisfied: rawpy>=0.16.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.17.1)
Requirement already satisfied: nltk>=3.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (3.7)
Requirement already satisfied: tqdm>=4.54.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (4.64.0)
Requirement already satisfied: imagecodecs in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (2022.2.22)
Requirement already satisfied: texttable~=1.6.3 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.6.4)
Requirement already satisfied: scikit-learn~=0.24.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (0.24.2)
Requirement already satisfied: progress>=1.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from openvino-dev) (1.6)
Requirement already satisfied: packaging>=14.3 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from nibabel>=3.2.1->openvino-dev) (21.3)
Requirement already satisfied: setuptools in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from nibabel>=3.2.1->openvino-dev) (59.5.0)
Requirement already satisfied: joblib in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from nltk>=3.5->openvino-dev) (1.1.0)
Requirement already satisfied: regex>=2021.8.3 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from nltk>=3.5->openvino-dev) (2022.4.24)
Requirement already satisfied: click in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from nltk>=3.5->openvino-dev) (8.1.3)
Requirement already satisfied: pytz>=2017.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from pandas~=1.1.5->openvino-dev) (2022.1)
Requirement already satisfied: python-dateutil>=2.7.3 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from pandas~=1.1.5->openvino-dev) (2.8.2)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from requests>=2.25.1->openvino-dev) (1.26.9)
Requirement already satisfied: idna<4,>=2.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from requests>=2.25.1->openvino-dev) (3.3)
Requirement already satisfied: certifi>=2017.4.17 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from requests>=2.25.1->openvino-dev) (2021.10.8)
Requirement already satisfied: charset-normalizer~=2.0.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from requests>=2.25.1->openvino-dev) (2.0.12)
Requirement already satisfied: tifffile>=2019.7.26 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from scikit-image>=0.17.2->openvino-dev) (2022.4.28)
Requirement already satisfied: imageio>=2.4.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from scikit-image>=0.17.2->openvino-dev) (2.9.0)
Requirement already satisfied: PyWavelets>=1.1.1 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from scikit-image>=0.17.2->openvino-dev) (1.3.0)
Requirement already satisfied: threadpoolctl>=2.0.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from scikit-learn~=0.24.1->openvino-dev) (3.1.0)
Requirement already satisfied: huggingface-hub<1.0,>=0.1.0 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from transformers>=4.5->openvino-dev) (0.5.1)
Requirement already satisfied: sacremoses in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from transformers>=4.5->openvino-dev) (0.0.53)
Requirement already satisfied: filelock in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from transformers>=4.5->openvino-dev) (3.6.0)
Requirement already satisfied: typing-extensions>=3.7.4.3 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from huggingface-hub<1.0,>=0.1.0->transformers>=4.5->openvino-dev) (3.7.4.3)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from packaging>=14.3->nibabel>=3.2.1->openvino-dev) (2.4.7)
Requirement already satisfied: six>=1.5 in /media/nobu/NTFSUbuWin/openvino/openvino_env/lib/python3.8/site-packages (from python-dateutil>=2.7.3->pandas~=1.1.5->openvino-dev) (1.15.0)
(openvino_env) nobu@nobu-IdeaPad-3-15IIL05:/media/nobu/NTFSUbuWin/openvino/openvino_notebooks$ 

```
